php-checks/src/test/resources/checks/security/XxeCheck.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+use Cake\Utility\Xml;
```

Comment:
The check does not take the namespace into account. You, unfortunately, have to request the symbol of the class member access to check for the FQN if it's Cake\Utility\Xml. Right now, your implementation fails on
```
$xml = Cake\Utility\Xml::build($content, ['loadEntities' => true]);
```